cql3/restrictions/partition_key_restrictions.cc

```diff
@@ -47,5 +47,10 @@ const expr::single_column_restrictions_map& partition_key_restrictions::get_sing
     const {
     return _single_column_partition_key_restrictions;
 }
+
+// Check if there are no partition key restrictions.
+bool partition_key_restrictions::partition_key_restrictions_is_empty() const {
+    return is_empty_restriction(_partition_restrictions);
```

Comment:
How does the original partition_key_restrictions handle a restriction that is not supported by read_command?

Does it return true or false here? If it returns false, what is it useful for?

Reply:
It looks like it's only used here:
https://github.com/scylladb/scylladb/blob/c5285bcb143a34b9bca6a6d167624237c994052d/cql3/statements/select_statement.cc#L868

I just copied the functionality to be the same as it was in `statement_restrictions`. Hopefully later we will be able to rewrite these conditions in a simpler way.

Reply:
Ok.